packages/react-ui/internal/Calendar/Calendar.tsx

```diff
@@ -212,7 +212,7 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
     const positions = this.getMonthPositions();
     const wrapperStyle = { height: themeConfig(this.theme).WRAPPER_HEIGHT };
     return (
-      <div ref={this.refRoot} className={styles.root(this.theme)}>
+      <div ref={this.refRoot} className={styles.root(this.theme)} data-tid="calendar-content">
```

Comment:
Значение только немного подправлю, в соответствии с нашим [соглашением](https://github.com/skbkontur/retail-ui/issues/1912). 

```suggestion
      <div ref={this.refRoot} className={styles.root(this.theme)} data-tid="Calendar">
```